CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 * Add support for Ingress v1 in Kubernetes 1.19 and newer
 * Add support for Kafka 2.6.1
 * List topics used by a Kafka Connect connector in the `.status` section of the `KafkaConnector` custom resource
+* Bump Cruise Control to v2.5.37 for Kafka 2.7 support. Note this new version of Cruise Control uses `log4j2` and is supported by dynamic logging configuration. However, existing `log4j` configurations must be updated to `log4j2` configurations.
```

Comment:
There is also another `Updated Cruise Control to version 2.5.32` message above which should be removed.